src/Compilers/CSharp/Test/Semantic/Semantics/TargetTypedObjectCreationTests.cs

```diff
@@ -3950,15 +3950,19 @@ static void Main()
             var node = tree.GetCompilationUnitRoot().DescendantNodes().OfType<ExpressionStatementSyntax>().Single();
             int nodeLocation = node.Location.SourceSpan.Start;
 
-            var modifiedNode = (ExpressionStatementSyntax)SyntaxFactory.ParseStatement("M(new());");
-            Assert.True(model.TryGetSpeculativeSemanticModel(nodeLocation, modifiedNode, out var speculativeModel));
+            var modifiedNode = (ExpressionStatementSyntax)SyntaxFactory.ParseStatement("M(new());", options: TargetTypedObjectCreationTestOptions);
```

Comment:
@333fred This is the fix. We were parsing with the default lang version and the node had an error.